kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/controller/DefaultRestControllerImpl.java

```diff
@@ -212,9 +212,7 @@ public KieServerSetup update(KieServerStateInfo serverTemplateUpdate) {
                 }
             }
 
-        } else {
-            logger.warn("No controllers found to update to new status {}", serverTemplateUpdate);
```

Comment:
Why this warning is no longer needed?

Reply:
it is too verbose and update operation is something that server does not need to received. this is just to solve an HA problem